Garage 2.0/Controllers/MembersController/MembersController.cs

```diff
@@ -25,10 +25,21 @@ public async Task<IActionResult> Index()
         {
             return View(await _context.Member.ToListAsync());
         }
-        public async Task<IActionResult> MemberOverviewIndex(string sortOrder, int? pageNumber = 1)
+        public async Task<IActionResult> MemberOverviewIndex(string sortOrder, string currentFilter, string ssn,int? pageNumber)
         {
             ViewData["CurrentSort"] = sortOrder;
             ViewBag.NameSortParm = String.IsNullOrEmpty(sortOrder) ? "FirstName_desc" : "";
+            if (ssn != null)
+            {
+                pageNumber = 1;
+            }
+            else
+            {
+                ssn = currentFilter;
+            }
```

Comment:
Could you add a the viewBag.Button here? 
I only want the button on the end of the memberOverviewIndex page to show when the user has done a search.

Reply:
i removed all extra viewbags because sorting and searching are interconnected

Reply:
i can show demo. then you can understand